docs/guide/Configs/Encoding.md

```diff
@@ -0,0 +1,81 @@
+# Encoding
```

Comment:
We have a new system documentation based on DocFX (see https://github.com/dotnet/BenchmarkDotNet/pull/781, https://github.com/dotnet/BenchmarkDotNet/pull/783). I'm working on it right now, so I can update docs/samples after merge.